### PR TITLE
Player/Path: Activate Cooldown Timer, DB Fixes, and Service Token costs

### PR DESCRIPTION
**Pull request by kirmmin**

### CODE REVIEW

File: Source/NexusForever.WorldServer/Game/Static/GenericError.cs [hunk 1]
```diff
@@ -0,0 +1,168 @@
+﻿namespace NexusForever.WorldServer.Game.Static
+{
+    public enum GenericError
```
Comment (Fabi) [inline Source/NexusForever.WorldServer/Game/Static/GenericError.cs]: Oh please. Go for numbers on all element

Comment (kirmmin) [reply Source/NexusForever.WorldServer/Game/Static/GenericError.cs]: Okaaaaaay 😛

File: Source/NexusForever.WorldServer/Game/Entity/PathManager.cs [hunk 2]
```diff
@@ -295,12 +292,17 @@ private void SendPathLogPacket()
...
+                PathProgress = paths.Values.Select(p => p.TotalXp).ToArray(),
                 PathUnlockedMask = GetPathUnlockedMask(),
-                ActivateTimer = 0 // TODO: Need to figure out timestamp calculations necessary for this value to update the client appropriately
+                TimeSinceLastActivateInDays = GetCooldownTime() // TODO: Need to figure out timestamp calculations necessary for this value to update the client appropriately
             });
         }
 
+        private float GetCooldownTime()
+        {
+            return (float)DateTime.Now.Subtract(player.PathActivatedTime).TotalDays * -1;
```
Comment (Therzok) [inline Source/NexusForever.WorldServer/Game/Entity/PathManager.cs]:
```suggestion
            return (float)DateTime.UtcNow.Subtract(player.PathActivatedTime).TotalDays * -1;
```

**Comment by kirmmin** [comment]
Rebased today to the latest `develop` branch.

**Comment by kirmmin** [comment]
Rebased today to the latest `develop` branch.

Also refactored the use of `PrerequisiteId` to the right format.

**Comment by kirmmin** [comment]
Rebased and updated today.

I also added Service Token costs using the GameFormulas.

**Pull request merged**